src/plugins/console/public/application/models/sense_editor/sense_editor.test.js

```diff
@@ -19,6 +20,13 @@ const { collapseLiteralStrings } = XJson;
 
 describe('Editor', () => {
   let input;
+  global.URL = URL;
+  global.window = Object.create(window);
+  Object.defineProperty(window, 'location', {
+    value: {
+      origin: 'http://localhost:5620',
+    },
+  });
```

Comment:
We are changing global variables that could affect other tests, I think will be safer to move this to a `beforeAll` instead and add another `afterAll` hook to clean up the mocks we added to the global state.

Reply:
thank you,added